bot/cogs/Calculator.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import re
+import discord
+import discord.ext.commands as commands
+
+log = logging.getLogger(__name__)
+
+
+class Calculator(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+        self._last_member = None
+        self.operators = [
+            {'symbol': '+', 'precedence': 0, 'assoc': 'L'},
+            {'symbol': '-', 'precedence': 0, 'assoc': 'L'},
+            {'symbol': '*', 'precedence': 1, 'assoc': 'L'},
+            {'symbol': '/', 'precedence': 1, 'assoc': 'L'},
+            {'symbol': '^', 'precedence': 2, 'assoc': 'R'},
+        ]
+
+    @commands.command()
+    async def calc(self, ctx, *args):
+        await ctx.send('{} arguments: {}'.format(len(args), ', '.join(args)))
```

Comment:
py doc comments explaining how to use the command need to be added, discord.py will automatically read those and provide that info when you do !help calc. If you have questions about how to format them or what pydoc comments are ping me in the server